Taskfile.yml

```diff
@@ -35,24 +36,25 @@ tasks:
       - echo "GOOGLE_OAUTH_CLIENT_ID=__REPLACE__.apps.googleusercontent.com" >> deployment/local-dev/local.env
       - echo "GOOGLE_OAUTH_CLIENT_SECRET=__REPLACE__" >> deployment/local-dev/local.env
       - kubectl --context=docker-desktop apply -f deployment/setup/namespace.yaml
+      - docker build -t local-dev-init data/
   setup-win:
     platforms: [ windows ]
     cmds:
       - cmd: powershell "if (test-path {{.HG_REPO_DIR}}/sena-3) { rm -r {{.HG_REPO_DIR}}/sena-3 }"
         ignore_error: true
         silent: true
-      - powershell -Command "Invoke-WebRequest 'https://drive.google.com/uc?export=download&id=1I-hwc0RHoQqW774gbS5qR-GHa1E7BlsS' -OutFile sena-3.zip"
-      - powershell -Command "Expand-Archive sena-3.zip -DestinationPath {{.HG_REPO_DIR}}"
-      - powershell rm sena-3.zip
+      - powershell -Command "Invoke-WebRequest 'https://drive.google.com/uc?export=download&id=1I-hwc0RHoQqW774gbS5qR-GHa1E7BlsS' -OutFile {{.DATA_DIR}}/sena-3.zip -Resume"
+      - powershell -Command "Expand-Archive {{.DATA_DIR}}/sena-3.zip -DestinationPath {{.HG_REPO_DIR}}"
+      - powershell -Command "Invoke-WebRequest 'https://drive.usercontent.google.com/download?export=download&id=1Jk-eSDho8ATBMS-Kmfatwi-MWQth26ro&confirm=t' -OutFile {{.DATA_DIR}}/elawa.zip -Resume"
```

Comment:
is there a reason we're not unpacking the elawa project too?

Reply:
Just that I didn't want to change the behavior. And come to think of it, why are we unpacking the sena-3.zip file here? That doesn't make sense any more; it's an artifact of when we used docker compose for local dev, but it's useless now that we've switched to Skaffold. So I'll just delete the unpacking of sena-3.

Reply:
Commit 0df4db3c removes unpacking the sena-3 repo, which doesn't make any sense now that we're using Skaffold. This should be resolved now.